sdk/src/main/java/io/opentelemetry/sdk/tags/TaggerSdkProvider.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.tags;
+
+import io.opentelemetry.tags.Tagger;
+import io.opentelemetry.tags.spi.TaggerProvider;
+
+/**
+ * {@code Tagger} provider implementation for {@link io.opentelemetry.tags.spi.TaggerProvider}.
+ *
+ * <p>This class is not intended to be used in application code and it is used only by {@link
+ * io.opentelemetry.OpenTelemetry}.
+ */
+public class TaggerSdkProvider implements TaggerProvider {
```

Comment:
Should all of the providers implement a common interface?  I realize we can't use the JDK one since we're not targeting java 8, but maybe add our own?

Reply:
Maybe consider autovalue https://github.com/google/auto/tree/master/service

Reply:
re auto: there is a test which verifies that provider is registered.

@tylerbenson I think it's an API change, do you want to create a ticket for it? I am fine with having providers as different explicit classes.

Reply:
Ok, it was just an idea... I won't worry about it.